crates/sui-json-rpc-api/src/lib.rs

```diff
@@ -104,6 +104,14 @@ pub trait RpcFullNodeReadApi {
         object: ObjectID,
     ) -> RpcResult<Vec<(GatewayTxSeqNumber, TransactionDigest)>>;
 
+    #[method(name = "getTransactionsByMoveFunction")]
+    async fn get_transactions_by_move_function(
+        &self,
+        package: ObjectID,
+        module: String,
+        function: String,
```

Comment:
Can we make these two arguments optional? i.e., people can just fetch all transactions by package names

Reply:
I think it should be fine for module to be optional, we should support that IMO.

As for function, I wonder if it's really needed.  We actually have removed support for functions in events, as after diving into details it became confusing, especially because for a specific Move event there is no corresponding function emitted.